### PR TITLE
Story/ite 58/all polls tabs fe

### PR DESCRIPTION
**Pull request by akhilarjun3-sureify**
Screenshot from 2023-07-03 13-19-45
Screenshot from 2023-07-03 13-19-52
Screenshot from 2023-07-03 13-19-59

### CODE REVIEW

**Review by maniknr-sureify — DISMISSED** [review]
change All polls to polls in sidebar, Follow proper naming for tabs Active polls,Ended polls,upcoming polls

**Comment by maniknr-sureify** [comment]
Remove conflicts

**Review by rameshguptha-sureify — APPROVED** [review]

**Review by harshavardhan-sureify — APPROVED** [review]

File: src/components/routes/AppRoutes.jsx [hunk 1]
```diff
@@ -40,13 +40,10 @@ const AppRoutes = () => {
...
+                element={<Polls  />}
               />
-              <Route
-                path="activepolls"
-                element={<Polls type={"activePolls"} />}
-              />
-              <Route path="activepolls/:id" element={<ViewSinglePoll />} />
+              
+              
               <Route path="allpolls/:id" element={<ViewSinglePoll />} />
```
Comment (abhishekdundigalla-sureify) [inline src/components/routes/AppRoutes.jsx:48]: please format this component code in a right way without any extra lines.

**Review by abhishekdundigalla-sureify — CHANGES_REQUESTED** [review]
Complete the required changes

**Review by Madhukar-Sureify — APPROVED** [review]

File: src/components/routes/AppRoutes.jsx (hunk 1 above)
Comment (akhilarjun3-sureify) [reply src/components/routes/AppRoutes.jsx:48]: changed

**Review by abhishekdundigalla-sureify — APPROVED** [review]

**Pull request merged**